project/Build.scala

```diff
@@ -573,7 +601,6 @@ object Build extends sbt.Build {
           normalizedName := "sbt-scalajs",
           name in bintray := "sbt-scalajs-plugin", // "sbt-scalajs" was taken
           sbtPlugin := true,
-          scalaVersion := "2.10.6",
```

Comment:
Yes, it does. And that allows `sbteclipse` (and IntelliJ, apparently) to generate a somewhat OK-ish project. Without that it would just blow up with dependency resolution hell.

This does not make actual building harder. It was always necessary to do `++2.10.6` before compiling the sbt plugin anyway, because of the dependent projects (tools and so on).